exchange/exchange.go

```diff
@@ -151,9 +151,15 @@ func (e *exchange) HoldAuction(ctx context.Context, bidRequest *openrtb.BidReque
 	adapterBids, adapterExtra, anyBidsReturned := e.getAllBids(auctionCtx, cleanRequests, aliases, bidAdjustmentFactors, blabels, conversions)
 
 	if anyBidsReturned {
-		bidCategory, adapterBids, err := applyCategoryMapping(ctx, requestExt, adapterBids, *categoriesFetcher, targData)
-		if err != nil {
-			return nil, fmt.Errorf("Error in category mapping : %s", err.Error())
+
+		var bidCategory map[string]string
+		//If includebrandcategory is present in ext then CE feature is on.
+		if requestExt.Prebid.Targeting != nil && requestExt.Prebid.Targeting.IncludeBrandCategory != nil {
+			var err error
```

Comment:
Veronica apparently there's no test that enters your newly written if statement. The following screenshot was taken from `coverage.html` a file generated using golang's default coverage tool.
```
╰─➤  go tool cover -html=c.out -o coverage.html
```

Reply:
![image](https://user-images.githubusercontent.com/15863840/65284949-1da43480-db09-11e9-8c62-8601c6d44246.png)

Reply:
@guscarreon unit test added. Can you please check if coverage changed?